### ISSUE TITLE
[enh] Remove dbg logs

### ISSUE THREAD

**Issue by zingwin** [Canardoux/tau#528]
我看源码，很多类似这样的日志输出。是否可以添加一个参数一键关闭，谢谢。
   print('FS:---> openAudioSession ');

**Comment by Larpoux** [comment]
?

**Comment by zingwin** [comment]
> ?

i need delete flutter_sound package log output
when i debug app, flutter_sound print too much log
Can you provide a switch to turn off the log output in the library with one click

thank you!

**Comment by Larpoux** [comment]
Yes @zingwin .

I know that it is something that we must do soon.
It will be fixed.

**Comment by Larpoux** [comment]
Registered in [the Flutter Sound Project](https://github.com/dooboolab/flutter_sound/projects/3)

**Comment by kavehmb** [comment]
any progress on removing the "print" logs?

**Comment by Larpoux** [comment]
Oh! yes 👍 
I began one week ago. I will use the **Logger plugin**, so that you will be able to set the Logs level you are interested by.

Unfortunately I had a problem with something terrible :  the new "Null Safety" option of dart.
Null Safety is becoming a terrible mess. This is a major breaking change in Dart, and everybody will have to do many changes in their code. Very bad.

I think that I will have to manage two releases of Flutter Sound. So for each version I will have : 
- Full With Null Safety
- Lite With Null Safety
- Full Without Null Safety
- Lite Without Null Safety

4 releases. A nighmare 👎 

And I have also to do user support and actually I spend too much time on that.
There is some bugs which need to be fixed before new developments.
And finally I do not tell you that I have another project for which it ids urgent that I work on.

I have never been able to manage 2 things at the same time.
But if you look [to Roadmap](https://github.com/Canardoux/tau/projects/3) you will see that I have 12 tasks in progress.
This is so many that I stop working.

I need contributions from other developers. Things are becoming too heavy for me.

**Comment by Larpoux** [comment]
But your post is good for me. It re-assure me that some Tasks In-Progress I am working on are really needed by some users.

**Comment by Larpoux** [comment]
Things are funny. During 3 weeks there had been nobody on the Flutter Sound front.
No issues, no requests, no help needed. Nothing.
And I worried to have lost all my users. I worried that I so badly worked that no more Flutter Sound users.

And suddenly every body are requesting something from me. And I come overloaded.

All things considered, I am happier to work for some users than working for nobody.
